pages/common/hg.md

```diff
@@ -0,0 +1,19 @@
+# hg
+
+> Provides an interface for Mercurial, a distributed source control management system.
```

Comment:
How about `Command line interface for Mercurial ..` ?

Reply:
I didn't know if `Command line` was necessary as it is used on the command line. :thinking: But ok.

Reply:
Simply saying "interface" might leave some doubt as to what kind of interface. Hence, to be extra clear, we can say 'command line interface'